internal/pkg/service/cli/cmd/cmd.go

```diff
@@ -243,6 +243,7 @@ func NewRootCommand(stdin io.Reader, stdout io.Writer, stderr io.Writer, osEnvs
 	root.addAlias(`r`, `template repository`)
 	root.addAlias(`repo`, `template repository`)
 	root.addAlias(`table`, `remote table`)
+	root.addAlias(`create-table`, `remote create table`)
```

Comment:
I can't use `table create` , because we are using alias `table`. 

I can't think of a good name for the alias.

Reply:
:thinking: but it doesn't match the naming style of other commands.

@hosekpeter I suggest to move code/command, to `remote/table/create/cmd.go`, so the command `remote table create` will be implemented, and then create an alias for the old command:

```
root.addAlias(`remote create table`, `remote table create`)
```

----------

Sorry, I don't know about this issue.

Reply:
The goal is: if someone has `remote create table` in some script, it must remain functional.